web/client/plugins/ContextExport.jsx

```diff
@@ -20,7 +20,8 @@ import { onContextExport } from "../actions/contextcreator";
 import { toggleControl } from "../actions/controls";
 import Message from "../components/I18N/Message";
 import Button from "../components/misc/Button";
-const isEnabled = createControlEnabledSelector("export");
+const EXPORT_CONTROL = "export-context";
```

Comment:
can this be used everywhere instead of hardcoding it? maybe it can be moved to a utils

Reply:
Sure. Probably to ControlUtils